packages/core/src/controllers/AccountController.ts

```diff
@@ -144,6 +146,14 @@ export const AccountController = {
     ChainController.setAccountProp('profileImage', profileImage, chain)
   },
 
+  setEmail(email: AccountControllerState['email'], chain: ChainNamespace) {
+    ChainController.setAccountProp('email', email, chain)
+  },
+
+  setUsername(username: AccountControllerState['username'], chain?: ChainNamespace) {
+    ChainController.setAccountProp('username', username, chain)
+  },
```

Comment:
Tricky case, cause they should be updated on all namespaces but not all namespaces might support auth wallet